lib/glimesh/resolvers/streams_resolver.ex

```diff
@@ -38,13 +38,23 @@ defmodule Glimesh.Resolvers.StreamsResolver do
     {:error, "Access denied"}
   end
 
+  def end_stream(_parent, %{stream_id: stream_id}, %{context: %{is_admin: true}}) do
+    stream = Streams.get_stream!(stream_id)
+
+    Streams.end_stream(stream)
+  end
+
   def end_stream(_parent, %{channel_id: channel_id}, %{context: %{is_admin: true}}) do
```

Comment:
I still think it's dangerous to allow ending of arbitrary stream instances by channel ID - if we allow this, we ought to plumb down some notification to anyone listening (FTL instances) that the stream IDs they're operating on have been invalidated so they don't just chug along happily not knowing that the service has decided that they're gone.